pkg/daemon/daemon.go

```diff
@@ -8,6 +8,10 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
failing verify here: 

`pkg/daemon/daemon.go:11: File is not `goimports`-ed (goimports)`

Reply:
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_machine-config-operator/2686/pull-ci-openshift-machine-config-operator-master-verify/1467926127505838080